eva_submission/ENA_submission/upload_to_ENA.py

```diff
@@ -87,3 +95,39 @@ def parse_ena_receipt(self, ena_xml_receipt):
             self.error('Cannot parse ENA receipt: ' + ena_xml_receipt)
             results['errors'].append('Cannot parse ENA receipt: ' + ena_xml_receipt)
         return results
+
+
+class ENAUploaderAsync(ENAUploader):
+
+    def upload_xml_files_to_ena(self):
+        webin_file = self.converter.create_single_submission_file()
+        file_dict = {
+            'file': (os.path.basename(webin_file), get_file_content(webin_file), 'application/xml'),
+        }
+        response = self._post_xml_file_to_ena(cfg.query('ena', 'submit_async'), file_dict)
+        json_data = response.json()
+        if 'submissionId' in json_data:
+            xml_link = [link_dict['href'] for link_dict in json_data['links'] if link_dict['rel'] == 'poll-xml'][0]
+            self.results['submissionId'] = json_data['submissionId']
+            self.results['poll-links'] = xml_link
+            self.monitor_results()
+        else:
+            self.results['errors'] = f'{json_data["status"]}: {json_data["error"]} - {json_data["message"]}'
+
+    def monitor_results(self, timeout=3600, wait_time=30):
+        xml_link = self.results['poll-links']
+        response = requests.get(xml_link, auth=HTTPBasicAuth(cfg.query('ena', 'username'), cfg.query('ena', 'password')))
```

Comment:
Should this one be retried as well?  (I'm starting to wonder if there's a better way of retrying our requests instead of writing these little wrapper methods everywhere, e.g. something in pyutils.)

Reply:
Could do but it is a very small query that is much less likely to fail. 
I also agree that we could built some retry-able  `requests` in pyutils